lib/hawkular/operations/operations_api.rb

```diff
@@ -105,7 +127,8 @@ def connect
         client.on(:message, once: true) do |msg|
           parsed_message = msg.data.to_msg_hash
 
-          logger.log("Sent WebSocket message: #{parsed_message}")
+          # Not working with websocket-vcr
```

Comment:
Maybe we should remove that session_id altogether (if could be used in the future we could fill an issue to seek alternatives in a separate PR).
Was this broken before or is a consequence of something new?